### ISSUE TITLE
Remove option to migrate :down

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#1081]
This no longer works:

```clojure
(use 'metabase.db)
(migrate @jdbc-connection-details :down)
```
It throws a
```
liquibase.exception.RollbackImpossibleException
No inverse to liquibase.change.core.RawSQLChange created
```

I think we need to write an inverse migration for the raw SQL migration introduced recently.

**Comment by camsaul** [comment]
If we're not going to fix this maybe we should remove the option

**Comment by agilliland** [comment]
yeah, that's fine with me.  at this point in time i don't think it's practical to try and support this.

**Comment by camsaul** [comment]
Ok, issue has been renamed

**Comment by salsakran** [comment]
@camsaul how bad is the state of migrations these days? I.e. are any of the data migrations reversible? How many of the raw sql ones are?

**Comment by camsaul** [comment]
With Liquibase you need to write a reverse migration by hand when you write a raw SQL migration in order to be able to reverse it. We're missing that in one or more migrations. However, I don't think there's been any new raw SQL migrations since this issue was opened. So all the recent migrations should be reversible (Liquibase can automatically reverse non-SQL migrations). 

`(migrate :down)` is still broken, because we still can't rollback *every* migration. But that's not something we'd want to do, anyway. We have another command, `(migrate :down-one)` that just rolls back the most recent migration, which should suffice.

So long story short it probably makes sense to take `(migrate :down)` out since it is broken and wouldn't be particularly useful even if it worked.
